repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/publicationlandingpage-main.ftl

```diff
@@ -157,47 +158,12 @@
                         <span>Updated:</span> ${getDefaultFormattedDate(document.updatedDate)}
                     </div>
 
-                    <#--  Publication type  -->
-                    <div class="hee-card--details__item">
-                        <span>Publication Type:</span>
-                        <#if publicationListingPageURL?has_content>
-                            <a href=${publicationListingPageURL}?publicationType=${document.publicationType}>${publicationTypeMap[document.publicationType]}</a>
-                        <#else>
-                            ${publicationTypeMap[document.publicationType]}
-                        </#if>
-                    </div>
 
-                    <#--  Publication professions  -->
-                    <#if document.publicationProfessions?has_content>
-                        <div class="hee-card--details__item">
-                            <span>Professions:</span>
-                            <#if publicationListingPageURL?has_content>
-                                <#list document.publicationProfessions as profession>
-                                    <a href=${publicationListingPageURL}?publicationProfession=${profession}>${publicationProfessionMap[profession]}</a><#sep>, </#sep>
-                                </#list>
-                            <#else>
-                                <#list document.publicationProfessions as profession>
-                                    ${publicationProfessionMap[profession]}<#sep>, </#sep>
-                                </#list>
-                            </#if>
-                        </div>
-                        </#if>
-
-                    <#--  Publication topics  -->
-                    <#if document.publicationTopics?has_content>
-                        <div class="hee-card--details__item">
-                            <span>Topics:</span>
-                            <#if publicationListingPageURL?has_content>
-                                <#list document.publicationTopics as topic>
-                                    <a href=${publicationListingPageURL}?publicationTopic=${topic}>${publicationTopicMap[topic]}</a><#sep>, </#sep>
-                                </#list>
-                            <#else>
-                                <#list document.publicationTopics as topic>
-                                    ${publicationTopicMap[topic]}<#sep>, </#sep>
-                                </#list>
-                            </#if>
-                        </div>
-                    </#if>
+                    <#-- partial -->
+                    <@publicationPartialInfo publicationListingPageURL=publicationListingPageURL
```

Comment:
Could we apply the `default` (`!`) operator for `publicationListingPageURL` to avoid the following error:
```
[WARNING] [talledLocalContainer] %d{HH:mm:ss} WARN  [HstFreemarkerServlet] When calling macro "publicationPartialInfo", required parameter "publicationListingPageURL" (parameter #1) was specified, but had null/missing value.
[INFO] [talledLocalContainer]
[INFO] [talledLocalContainer] ----
[INFO] [talledLocalContainer] Tip: If the parameter value expression on the caller side is known to be legally null/missing, you may want to specify a default value for it with the "!" operator, like paramValue!defaultValue.
[INFO] [talledLocalContainer] ----
[INFO] [talledLocalContainer]
[INFO] [talledLocalContainer] ----
[INFO] [talledLocalContainer] FTL stack trace ("~" means nesting-related):
[INFO] [talledLocalContainer] 	- Failed at: #macro publicationPartialInfo publica...  [in template "webfile:/freemarker/hee/macros/publication-partial-info.ftl" in macro "publicationPartialInfo" at line 1, column 1]
[INFO] [talledLocalContainer] 	- Reached through: @publicationPartialInfo publicationLi...  [in template "webfile:/freemarker/hee/catalog/publicationlandingpage-main.ftl" at line 163, column 21]
[INFO] [talledLocalContainer] ----. To see the stack trace, set 'org.hippoecm.hst.servlet.HstFreemarkerServlet' log-level to debug in log4j configuration or runtime via the logging servlet
```

Reply:
Will do